### ISSUE TITLE
Spam -10 hit doesn't register on -1 comments

### ISSUE THREAD

**Issue by soytakyon** [SoylentNews/rehash#211]
Comments that have already been modded -1 do not incur a -10 subnet karma penalty for a spam mod, even though the spam mod is applied by the system to the list of mods.

**Comment by TheMightyBuzzard** [comment]
This particular bug should properly be classified as "why the hell are we allowing spam mods to something already moderated down to -1".

**Comment by marty-b** [comment]
<p>So, assume a comment which had (possibly incorrectly, but, meh) previously been moderated "-1 Troll" or "-1 Flamebait" and is now at a total moderation of "-1".
<p>Then, someone comes along, notices the comment truly IS spam, and wishes to not only flag it as such, but also to invoke the consequences of a spam mod against the comment poster: a -10 Karma hit and moderation banning.  If I understand this bug report correctly, a user is unable to apply a spam mod when a comment is already at -1.  It would require an upmod by someone before it could, then, be moderated as spam. Something seems wrong with that.
<p>I think the original report raises a valid point.

**Comment by TheMightyBuzzard** [comment]
Being modded Spam does not get you a moderation ban. Abusing the Spam mod gets you that.

**Comment by marty-b** [comment]
Yes, you are right -- I misremembered, and, apparently, misread the original bug report.
<p>Would help to have a link to the comment in question.
